doc/manual.rst

```diff
@@ -4562,18 +4566,22 @@ needs to fit the types of `except` branches, but the type of the `finally`
 branch always has to be `void`:
 
 .. code-block:: nim
+    :test: "nim c $1"
   from std/strutils import parseInt
 
   let x = try: parseInt("133a")
-          except: -1
+          except CatchableError, Defect: -1
```

Comment:
Might be a good opportunity to show better patterns, eg
`except CatchableError:`
or even
`except ValueError:`
which is is only possible of raise of parseInt AFAICT